.travis.yml

```diff
@@ -4,58 +4,38 @@ branches:
 os: osx
 matrix:
   include:
-    # Note: The ordering of the environments is optimized so that the complete matrix runs as quickly as possible,
-    # while also providing the first results as early as possible.
-    # At the moment, build times for each environment are roughly as follows:
-    # xcode7.3: 16 minutes
-    # xcode8.3: 7 minutes
-    # xcode9.2: 7 minutes
-    # xcode10.1: 6 minutes
-    # https://docs.travis-ci.com/user/reference/osx/#macos-version
-    # This data is based on the following build, which should be similar to most builds on the repo:
-    # https://travis-ci.org/pybee/rubicon-objc/builds/506412489
-    # Currently, Travis allows a maximum of two concurrent Mac builds per owner:
-    # https://blog.travis-ci.com/2017-09-22-macos-update
-    # With the following order, xcode7.3 will take up the first slot, and at the same time the three other
-    # environments use the second slot in sequence. xcode10.1 will finish first and provide some early results - most
-    # test failures happen in all environments and will be quickly reported this way.
-
     # OSX 10.11 El Capitan
     - osx_image: xcode7.3
+    # macOS 10.12 Sierra
+    - osx_image: xcode9.2
+    - osx_image: xcode8.3
```

Comment:
Minor thing: the rest of the version list is ordered from oldest to newest, so `xcode8.3` should come before `xcode9.2`.